.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -302,7 +302,7 @@ data class TestCoverage(
     }
 
     fun asName(): String =
-        "Test Coverage - ${testType.name.capitalize()} ${testJvmVersion.name.capitalize()} ${vendor.name.capitalize()} ${os.asName()}${if (withoutDependencies) " without dependencies" else ""}"
+        "${testType.name.capitalize()} ${testJvmVersion.name.capitalize()} ${vendor.name.capitalize()} ${os.asName()}${if (withoutDependencies) " without dependencies" else ""}"
```

Comment:
We should probably change the project name as well, right? Should we add `Test Coverage` at the end then? Or as description?

On a similar point, we could also remove the `Performance tests for` from the performance tests buckets.